src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,18 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer {
```

Comment:
Raw use of parameterized class 'MachineProducer'

Reply:
I think, we don't need parametrization here, because we use it in public interface MachineProducer<T extends Machine>